src/game/scenes/arena.c

```diff
@@ -1520,8 +1520,8 @@ int arena_create(scene *scene) {
     // Initial har data
     vec2i pos[2];
     int dir[2] = {OBJECT_FACE_RIGHT, OBJECT_FACE_LEFT};
-    pos[0] = vec2i_create(HAR1_START_POS, ARENA_FLOOR);
-    pos[1] = vec2i_create(HAR2_START_POS, ARENA_FLOOR);
+    pos[0] = vec2i_create(HAR1_START_POS / 256, ARENA_FLOOR / 256);
```

Comment:
I feel like the 2i should hold the full pseudo float, and we have functions that  divide by 256 to get the pixel coordinate when we need it

Reply:
I think we should do it all the right way and fix what breaks. I can assist if you want.